app/src/main/java/com/github/gotify/service/WebSocketConnection.java

```diff
@@ -171,7 +202,7 @@ public void onFailure(WebSocket webSocket, Throwable t, Response response) {
         void execute(String message);
     }
 
-    interface OnFailureCallback {
-        void execute(int minutesToTryAgain);
+    interface onNetworkFailureRunnable {
```

Comment:
Could you make the first character uppercased?
```suggestion
    interface OnNetworkFailureRunnable {
```